src/packages/@ncigdc/modern_components/ProjectCounts/ProjectCountsDataCategory.js

```diff
@@ -132,10 +153,62 @@ export default compose(
           style: { textAlign: 'right' },
         },
         {
+          key: 'case_count_meter',
+          title: (
+            <Link
+              pathname="/repository"
+              query={{
+                filters: makeFilter([
+                  {
+                    field: 'project.project_id',
+                    value: project.project_id,
+                  },
+                ]),
+                facetTab: 'files',
+                searchTableTab: 'files',
+              }}
+              title="Browse files"
+            >
+              <SampleSize n={totalCases} />
+            </Link>
+          ),
+          thStyle: {
+            width: 1,
+            textAlign: 'center',
+          },
+          style: { textAlign: 'left' },
+        },
+        {
           key: 'file_count',
           title: 'Files',
           style: { textAlign: 'right' },
         },
+        {
+          key: 'file_count_meter',
+          title: (
+            <Link
+              pathname="/repository"
+              query={{
+                filters: makeFilter([
+                  {
+                    field: 'project.project_id',
```

Comment:
same as above, `cases.project.project_id`

Reply:
note not `files` here. even though we make the filter on the files index here, project_id belongs to a case when determining which project a file belongs to